extensions/wrapper/wrapper/src/main/java/de/sovity/edc/ext/wrapper/api/ui/model/ContractDefinitionEntry.java

```diff
@@ -0,0 +1,40 @@
+/*
+ *  Copyright (c) 2022 sovity GmbH
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       sovity GmbH - initial API and implementation
+ *
+ */
+
+package de.sovity.edc.ext.wrapper.api.ui.model;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.Data;
+import org.eclipse.edc.spi.query.Criterion;
+
+import java.util.List;
+
+@Data
+@Schema(description = "Contract Definition Entry for Contract Definition Page")
+public class ContractDefinitionEntry {
+
+    @Schema(description = "Access Policy ID", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String accessPolicyId;
+
+    @Schema(description = "Contract Policy ID", requiredMode = Schema.RequiredMode.REQUIRED)
+    private String contractPolicyId;
+
+    @Schema(description = "Criteria for the contract", requiredMode = Schema.RequiredMode.REQUIRED)
+    private List<Criterion> criteria;
```

Comment:
No Core EDC Types should be leaked. We need to add our own objects and do the mapping.